public/locales/cs/common.json

```diff
@@ -38,9 +38,9 @@
     "movapp_goal_title": "<strong>Cíl Movapp.cz</strong>",
     "movapp_goal_description": "Cílem Movapp.cz je usnadnit dorozumění mezi Čechy a Ukrajinci. V první fázi byl vypublikován přehled základních česko-ukrajinských frází. V dalších fázích budeme vytvářet nástroje pro jejich procvičování a chceme se stát rozcestníkem pro další nástroje a materiály, které pomůžou s učením ukrajinštiny a češtiny.",
     "why_movapp_title": "<strong>Proč Movapp?</strong>",
```

Comment:
I don't think this is a very good pattern. It's fine now but it could cause problems in the future. Translations should not include HTML tags, the formatting information should be kept inside components where the translations are used. If we need to format something inside a translation, we should use the <0> </0> pattern we currently use for links. Translations should be kept short enough to keep that practical, i.e. one paragraph per translation. @met Can I try to restructure this? :)